File: LTUDDD_TLU/app/src/main/java/com/example/ltuddd_tlu/tuan4/tuan42_Java/tuan42MainActivity.java
package com.example.ltuddd_tlu.tuan4.tuan42_Java;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.content.Context;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.example.ltuddd_tlu.R;

public class tuan42MainActivity extends AppCompatActivity {

    Button btnGetData;
    TextView tvKQ;
    Context context = this;

    @SuppressLint("WrongViewCast")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tuan42_main);

        tuan42VolleyFN volleyFN = new tuan42VolleyFN();
        tvKQ = findViewById(R.id.tuan42TvKQ);
        btnGetData = findViewById(R.id.tuan42Btn);
        btnGetData.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                volleyFN.getJsonArrayOfObject(context,tvKQ);
            }
        });



    }
}

File: LTUDDD_TLU/app/src/main/java/com/example/ltuddd_tlu/tuan4/tuan42_Kotlin/tuan42MainActivity.kt
package com.example.ltuddd_tlu.tuan4.tuan42_Kotlin

import android.content.Context
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example.ltuddd_tlu.R

class tuan42MainActivity : AppCompatActivity() {

    var btn:Button? = null
    var tv:TextView? = null
    var context: Context = this
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tuan42_main2)

        btn = findViewById(R.id.tuan42Btn2)
        tv = findViewById(R.id.tuan42Tv2)

        val fn = tuan42VolleyKotlinFL()
        btn!!.setOnClickListener {
            fn.getJSON_ArrayObjects(context, tv!!)
        }

    }
}